meterfs/meterfs.c

```diff
@@ -565,7 +565,7 @@ int meterfs_allocateFile(const char *name, size_t sectorcnt, size_t filesz, size
 	if (recordsz > filesz || !recordsz)
 		return -EINVAL;
 
-	strncpy(hdr.name, name, sizeof(hdr.name));
+	memcpy(hdr.name, name, sizeof(name));
```

Comment:
what? :)
```c
sizeof(name) == sizeof(const char*) == sizeof(uintptr_t)
``` 
So... it will be 4 or 8 on 32-bit or 64-bit architectures - this is obviously wrong :)

Reply:
I still working on it :D